pyuvdata/uvdata/ms.py

```diff
@@ -119,6 +119,63 @@ def _write_ms_antenna(self, filepath):
             antenna_table.putcol("DISH_DIAMETER", ant_diam_table)
         antenna_table.done()
 
+    def _write_ms_feed(self, filepath):
+        """
+        Write out the feed information into a CASA table.
+
+        Parameters
+        ----------
+        filepath : str
+            path to MS (without FEED suffix)
+
+        """
+        if not casa_present:  # pragma: no cover
+            raise ImportError(no_casa_message) from casa_error
+
+        feed_table = tables.table(filepath + "::FEED", ack=False, readonly=False)
+
+        nfeeds_table = np.max(self.antenna_numbers) + 1
+        feed_table.addrows(nfeeds_table)
+
+        antenna_id_table = np.arange(nfeeds_table, dtype=np.int32)
+        feed_table.putcol("ANTENNA_ID", antenna_id_table)
+
+        beam_id_table = np.ones(nfeeds_table, dtype=np.int32)
+        feed_table.putcol("BEAM_ID", beam_id_table)
+
+        beam_offset_table = np.zeros((nfeeds_table, 2, 2), dtype=np.float64)
+        feed_table.putcol("BEAM_OFFSET", beam_offset_table)
+
+        feed_id_table = np.zeros(nfeeds_table, dtype=np.int32)
```

Comment:
I think this needs to be zeros for one of the arms ('x'?) and 1 for the other. It's hard to interpret from the docs (they're self-referential) but the docs do say:
> Feeds are numbered from 0 on each separate antenna